templates/haproxy-config

```diff
@@ -22,6 +23,7 @@ case "$1" in
               cp "$conf_fragment" "{{ LOAD_BALANCER_CONF_DIR }}/$fragment_name"
         flock -s "{{ LOAD_BALANCER_BACKENDS_DIR }}" \
               awk '{print tolower($1) " " $2}' "$map_fragment" > "{{ LOAD_BALANCER_BACKENDS_DIR }}/$fragment_name"
+        if command -v etckeeper &> /dev/null; then etckeeper commit; fi
```

Comment:
Could you please add a commit message?  It can be passed as parameter to `etckeeper commit`, e.g.

    etckeeper commit "Auto-commit after `haproxy-config apply`."

or similar.

Reply:
Ah yes, sure. I'm testing to see if tests even pass correctly with this first. As soon as they do I'll add it.